### PR TITLE
Add issue & PR templates

### PR DESCRIPTION
**Pull request by loikein**

### CODE REVIEW

**Comment by loikein** [comment]
Once these changes get approved, we can copy them over https://github.com/OpenSourceEconomics/.github/issues/1

File: configurations/rtd_environment.yml [hunk 1]
```diff
@@ -29,7 +29,7 @@ dependencies:
   - hypothesis
   - nbsphinx
   - sphinx_rtd_theme
-  - sphinxcontrib-bibtex
+  - sphinxcontrib-bibtex<2.0.0
```
Comment (peisenha) [inline configurations/rtd_environment.yml:32]: Why is this needed?

**Review by peisenha — APPROVED** [review]
Great.thanks! Just one little question below ...

File: configurations/rtd_environment.yml (hunk 1 above)
Comment (loikein) [reply configurations/rtd_environment.yml:32]: Seems that a recent update to sphinxcontrib breaks the workflow, and this is a quick fix so that the document builds. I can take a look at it later, if you want.
